Assets/Scripts/Level/TutorialTrigger.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using UnityEngine;
+
+namespace Assets.Scripts.Level
+{
+    /**
+     * Create an instance of TutorialTriggerGroups to populate each Trigger to speed up GetComponent calls
+     */
+    public class TutorialTrigger : MonoBehaviour

```

Comment:
you can add the custom attribute [RequireComponent] here to prevent any issues in Start()